### PR TITLE
feat: add in a page about all of our projects

### LINKED ISSUE #21: Page for `all-the-things`
We can add a page called, Projects and Platforms (at a path say, `/all-the-things` - This page will be a list of all the projects and how they are linked together, tech stack they use and a brief description. 

That way people get to "discover" projects they are interested in? This page should be the default CTA on the landing page (see #20) ?

### PR DESCRIPTION
**Pull request by a2937**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #21

This is a rough draft of the page so far. I'm not sure how much of a landing page it seems like at the minute, and I feel like I excessively used hyperlinks. I'm also working on adding all of our relevant projects.

### CODE REVIEW

File: src/content/docs/all-the-things.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+- [NextJS](https://nextjs.org/) : this React powered framework is responsible for handling our API.
+
+### [Our Web 3.0 Curriculum](https://web3.freecodecamp.org/)
+
+Thanks to sponsorships, freeCodeCamp offers specialized content that discusses blockchain and cryptocurrency. The source code can be found on [GitHub](https://github.com/freeCodeCamp/web3-site)
+
+### [Our forums](https://forum.freecodecamp.org/)
+
+Our forums where campers can discuss troubles with the challenges and various other topics related to development. The Discourse platform hosts all of the content and makes this experience possible.
```
Comment (majestic-owl448) [inline src/content/docs/all-the-things.md]: Related repo is the forum-theme one https://github.com/freeCodeCamp/forum-theme

File: src/content/docs/all-the-things.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+Our publicly hosted accumulation of scraped programming language documentation from various sources and provides it in one convenient easy to search place. The source code can be found on [GitHub](https://github.com/freeCodeCamp/devdocs/).
+
+This is powered by the [Ruby](https://www.ruby-lang.org/en/) programming language.
+
+### [Our Youtube Channel](https://www.youtube.com/@freecodecamp/videos)
+
+Here we host various tutorials generously brought in by our contributors. This is also home of the FreeCodeCamp podcast where Quincy interviews notable people in the programming industry and also discusses upcoming trends in the development world.
+
+[CamperChan](https://github.com/freeCodeCamp/camperchan)
```
Comment (majestic-owl448) [inline src/content/docs/all-the-things.md]:
```suggestion
### [CamperChan](https://github.com/freeCodeCamp/camperchan)
```

File: src/content/docs/all-the-things.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+- [MongoDB](https://www.mongodb.com/) : Our database of choice
+
+### [Learn to Code RPG](https://freecodecamp.itch.io/learn-to-code-rpg)
+
+A Renpy game that provides an introduction to various programming related concepts. The source code can be found on [GitHub](https://github.com/freeCodeCamp/LearnToCodeRPG/).
+
+### [Learn to Code RPG](https://freecodecamp.itch.io/learn-to-code-rpg)
+
+A Renpy game that provides an introduction to various programming related concepts. The source code can be found on [GitHub](https://github.com/freeCodeCamp/LearnToCodeRPG/).
```
Comment (majestic-owl448) [inline src/content/docs/all-the-things.md]: this is a duplicate

**Review by majestic-owl448 — COMMENTED** [review]
I was taking a look and I saw these things

File: src/content/docs/all-the-things.md (hunk 3 above)
Comment (a2937) [reply src/content/docs/all-the-things.md]: Thank you for pointing this out.

File: src/content/docs/all-the-things.md (hunk 2 above)
Comment (a2937) [reply src/content/docs/all-the-things.md]: Got it. I've applied the header now.

File: src/content/docs/all-the-things.md (hunk 1 above)
Comment (a2937) [reply src/content/docs/all-the-things.md]: Thank you. I was unaware it had a repository.

**Pull request closed without merging**